codesystems/CodeSystem-UKCore-PracticeSettingCode.xml

```diff
@@ -0,0 +1,412 @@
+  <copyright value="Copyright &#169; 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the &quot;License&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7&#174; FHIR&#174; standard Copyright &#169; 2011+ HL7 The HL7&#174; FHIR&#174; standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+  <caseSensitive value="true" />
+  <content value="complete" />
+	<concept>
+		<code value = "100" />
+		<display value = "General Surgery" />
+	</concept>
+	<concept>
+		<code value = "101" />
+		<display value = "Urology" />
+	</concept>
+	<concept>
+		<code value = "107" />
+		<display value = "Vascular Surgery" />
+	</concept>
+	<concept>
+		<code value = "110" />
+		<display value = "Trauma and Orthopaedics" />
+	</concept>
+	<concept>
+		<code value = "120" />
+		<display value = "Ear Nose and Throat" />
+	</concept>
+	<concept>
+		<code value = "130" />
+		<display value = "Ophthalmology" />
+	</concept>
+	<concept>
+		<code value = "140" />
+		<display value = "Oral Surgery" />
+	</concept>
+	<concept>
+		<code value = "141" />
+		<display value = "Restorative Dentistry" />
+	</concept>
+	<concept>
+		<code value = "142" />
+		<display value = "Paediatric Dentistry" />
+	</concept>
+	<concept>
+		<code value = "143" />
+		<display value = "Orthodontics" />
+	</concept>
+	<concept>
+		<code value = "145" />
+		<display value = "Oral and Maxillofacial Surgery" />
+	</concept>
+	<concept>
+		<code value = "146" />
+		<display value = "Endodontics" />
+	</concept>
+	<concept>
+		<code value = "147" />
+		<display value = "Periodontics" />
+	</concept>
+	<concept>
+		<code value = "148" />
+		<display value = "Prosthodontics" />
+	</concept>
+	<concept>
+		<code value = "149" />
+		<display value = "Surgical Dentistry" />
+	</concept>
+	<concept>
+		<code value = "150" />
+		<display value = "Neurosurgery" />
+	</concept>
+	<concept>
+		<code value = "160" />
+		<display value = "Plastic Surgery" />
+	</concept>
+	<concept>
+		<code value = "170" />
+		<display value = "Cardiothoracic Surgery" />
+	</concept>
+	<concept>
+		<code value = "171" />
+		<display value = "Paediatric Surgery" />
+	</concept>
+	<concept>
+		<code value = "191" />
+		<display value = "Pain Management (Retired 1 April 2004)" />
```

Comment:
Are we meant to be setting these as retired using a property:

    <property>
      <code value="status"/>
      <valueCode value="retired"/>
    </property>